lucille-core/src/main/java/com/kmwllc/lucille/stage/ComputeFieldSize.java

```diff
@@ -0,0 +1,55 @@
+package com.kmwllc.lucille.stage;
+
+import java.util.Iterator;
+import com.kmwllc.lucille.core.Document;
+import com.kmwllc.lucille.core.Stage;
+import com.kmwllc.lucille.core.StageException;
+import com.kmwllc.lucille.core.UpdateMode;
+import com.typesafe.config.Config;
+
+/**
+ * Computes size of given byte array field and adds a field with that size
+ * <br>
+ * Config Parameters -
+ * <br>
+ * <p>
+ * <b>source</b> (String) : The field containing the byte array
+ * </p>
+ * <p>
+ * <b>destination</b> (String) : The name of the field which will be added to store the size. Will overwrite value if field already
+ * exists
+ * </p>
+ */
+public class ComputeFieldSize extends Stage {
+
+  private final String source;
+  private final String destination;
+
+  public ComputeFieldSize(Config config) {
+    super(config, new StageSpec().withRequiredProperties("source", "destination"));
```

Comment:
There's an implicit convention in the stage package where a destination field is called "dest" and not "destination"
Let's follow that convention there.
I found one exception -- the Timestamp stage calls it "dest_field" but we should change that.